app/src/main/java/space/taran/arkshelf/domain/Link.kt

```diff
@@ -0,0 +1,10 @@
+package space.taran.arkshelf.domain
+
+import java.nio.file.Path
+
+data class Link(
+    var title: String,
+    var desc: String,
+    val imagePath: Path?,
+    val url: String
```

Comment:
Is it possible to normalize the URL? E.g. "www.duckduckgo.com" is the same as "duckduckgo.com". Also, if a website is accessible both by `https` and `http`, `https` should be preferred.